packages/next/package.json

```diff
@@ -70,6 +70,7 @@
   },
   "dependencies": {
     "@next/env": "12.1.1-canary.1",
+    "bytes": "3.1.1",
```

Comment:
Can we make this a `devDependency` and pre-compile it similar to our other dependencies? An example of this can be seen here https://github.com/vercel/next.js/blob/586616c4cb28b4cdac7b715c49b0ceea2e5079bc/packages/next/taskfile.js#L131 and should be imported from `next/dist/compiled/bytes`

Reply:
Perfect, was not sure how this was being handled.